tutorcairn/templates/cairn/apps/superset/superset_config.py

```diff
@@ -32,7 +36,9 @@
     "pt_BR": {"flag": "br", "name": "Brazilian Portuguese"},
     "ru": {"flag": "ru", "name": "Russian"},
     "ko": {"flag": "kr", "name": "Korean"},
+    "sk": {"flag": "sk", "name": "Slovak"},
     "sl": {"flag": "si", "name": "Slovenian"},
+    "nl": {"flag": "nl", "name": "Dutch"},
```

Comment:
Nice to get new languages! Please sort languages by key. Also, update the blob/dc57508... link above to point to version 3.0.1.